packages/shared/src/testing/vrt.ts

```diff
@@ -84,6 +84,7 @@ export const getVisualRegressionPropTableTester = (): VisualRegressionTester =>
 
 type VRTestOptions = TestOptions & {
   scenario?: (page: Page) => Promise<void>;
+  scaledComponents?: boolean;
```

Comment:
Can we rename this to something that sounds more like a boolean and from the perspective of configuring the test setup?